openshift/clowder/clowd-app.yaml

```diff
@@ -4,6 +4,15 @@ kind: Template
 metadata:
   name: automation-hub
 objects:
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: pulp-key
+    namespace: automation-hub
+  data:
+    database_fields.symmetric.key: |
+      DNmNdwgyZugTax9S64J0FITTr9IHPxbuoF1F1CGPr68=
```

Comment:
Our security teams runs monitoring for leaked keys.

To avoid this to be catched by the monitoring we have this file https://github.com/ansible/galaxy_ng/blob/master/.gitleaks.toml#L3

We need to add `openshift/clowder/clowd-app.yaml` to that list of paths.